common/lib/xmodule/xmodule/lti_module.py

```diff
@@ -132,7 +134,10 @@ class LTIFields(object):
         help=_(
             "Enter the URL of the external tool that this component launches. "
             "This setting is only used when Hide External Tool is set to False."
-            "<br />See " + DOCS_ANCHOR_TAG + " for more details on this setting."
+            "<br />See {docs_anchor_open}the edX LTI documentation{anchor_close} for more details on this setting."
```

Comment:
Rather than having an HTML linebreak in this string, it seems to me like it should be two separate, translated strings.